hack/e2e.sh

```diff
@@ -52,6 +52,8 @@ oc secrets link default ci-pull-secret --for=pull
 # https://docs.openshift.com/container-platform/4.3/monitoring/monitoring-your-own-services.html
 oc -n openshift-monitoring create configmap cluster-monitoring-config --from-literal='config.yaml={"techPreviewUserWorkload": {"enabled": true}}' -o yaml --dry-run=client > /tmp/cluster-monitoring-config.yaml
 oc apply -f /tmp/cluster-monitoring-config.yaml
+oc -n openshift-user-workload-monitoring create configmap user-workload-monitoring-config --from-literal='config.yaml=' -o yaml --dry-run=client > /tmp/cluster-user-workload-monitoring-config.yaml
```

Comment:
Did we want to lead with the doc URI here too, to match the 4.3 doc link above, and make it clear that this new way is a separate block?

Reply:
Done, PTAL